packages/react-dom/src/__tests__/ReactLegacyMount-test.js

```diff
@@ -248,7 +248,7 @@ describe('ReactMount', () => {
     expect(calls).toBe(5);
   });
 
-  // @gate !disableLegacyMode
+  // @gate !disableLegacyMode && classic
```

Comment:
Since this no-ops in every entry point except www-classic, these tests need the extra gate.